test/unit/test-layout.cpp

```diff
@@ -74,9 +74,9 @@ TEST(TypedLayoutTest, 1D)
 TEST(LayoutTest, OffsetVsRegular)
 {
   const auto layout =
-      RAJA::make_permuted_layout({6, 6}, RAJA::Perm<1, 0>::value);
```

Comment:
Nice fix, this caught me yesterday in some stuff I was doing with SAMRAI.

Reply:
This may be a compiler bug (https://bugs.llvm.org/show_bug.cgi?id=21629) because gcc and intel don't have a problem with this.  It's very annoying.